uo_sys_framework/ACREModule/cfgAttributeCategories/Object.hpp

```diff
@@ -94,5 +94,77 @@ class UO_FW_ACREOptions {
 			condition = "objectBrain";
 			defaultValue = "0";
 		};
+		class UO_FW_ACRE_RACK_ENABLED {
+			property = "UO_FW_ACRE_RACK_ENABLED";
+			displayName = "Add Rack to Object";
+			tooltip = "Allows objects to be given radio racks defined by ACRE settings";
+			control = "CheckBox";
+			expression = "_this setVariable ['%s',_value,true]; if (_value) then {[{CBA_missionTime > 1},{[_this] call UO_FW_FNC_addRack;},[_this],30,{(format [""(ACRE_SETUP_MODULE) [ERR]: CBA_missionTime COULD NOT BE FOUND AFTER 30 SECONDS, UO_FW_FNC_addRack TIMED OUT""]) call UO_FW_FNC_DebugMessage;}] call CBA_fnc_WaitUntilAndExecute;};";
+			condition = "objectVehicle + (object * (1-objectBrain)) + (1 - objectControllable)";
```

Comment:
Doesn't `condition = "objectVehicle"` cover all vehicles while excepting units? 

<https://community.bistudio.com/wiki/Eden_Editor:_Configuring_Attributes#Condition>

Reply:
Does not exclude from what I understand.

Reply:
All vanilla vehicle attributes in 3den use `condition = "objectVehicle"`

Reply:
```Condition 	        For Type   Description
objectBrain 	        Object 	   Object with simulation "soldier" or "UAVpilot"
objectControllable 	Object 	   Object with simulation "soldier"
objectAgent 	        Object     Object with non-empty agentTasks[], i.e., animals
objectVehicle 	        Object 	   Vehicle which can be boarded
objectSimulated 	Object 	   Object which is simulated (e.g., falls down when in the air)
objectDestructable 	Object 	   Indestructible object, i.e., when destrType config property set not DestructNo
logicModule 	        Logic 	   Logic which is a module (vehicleClass config property is "Modules")
objectHasInventoryCargo Object 	   All object which as an openable inventory

Reply:
Would that apply to modded vehicles? (AKA R.H.S vehicles which the functionality was created to address.)

Reply:
Yes, any vehicle which can be boarded. Modded vehicles inherit from existing vehicle classes that this accounts for